resources/views/meetups/index.blade.php

```diff
@@ -17,7 +17,7 @@
                         {{ $meetup->language }}
```

Comment:
Why did you changed chmods from 644 to 755 for so many files?

Reply:
I did not change the chmod and it seems to me that the last pull req has been changed on the master

Reply:
done i revert to chmod 644